clloader/scenarios/permutations.py

```diff
@@ -0,0 +1,68 @@
+from typing import Callable, List
+
+import torch
+
+from clloader.datasets import _ContinuumDataset
+from clloader.scenarios import TransformationIncremental
+
+from torchvision import transforms
+
+
+class PermutationTransform:
+    """Permutation transformers"""
+
+    def __init__(self, seed):
+        self.seed = seed
+        self.g_cpu = torch.Generator()
+
+    def __call__(self, x):
+        shape = list(x.shape)
+        x = x.view(-1)
+        # if seed is 0, no permutations
+        if self.seed != 0:
+            self.g_cpu.manual_seed(self.seed)
```

Comment:
Shouldn't the seed be set in the `__init__`?

Reply:
le code est un peu tiré par les cheveux mais c'est parce que j'ai pas acces à la taille des points de données avant le __ call __ ...